### PR TITLE
use unit id being sent instead of hard coded auid

### PR DESCRIPTION
**Pull request by HolzAndrew**
- [x ] Other

## Description of change
uses unit bid param rather than hard coding unit Id. Updates some tests accordingly.

### CODE REVIEW

**Comment by HolzAndrew** [comment]
@jasminenoack plz to review

File: modules/openxoutstreamBidAdapter.js [hunk 1]
```diff
@@ -83,7 +74,7 @@ function buildCommonQueryParamsFromBids(bids, bidderRequest) {
     tws: getViewportDimensions(isInIframe),
     be: 1,
     bc: bids[0].params.bc || `${BIDDER_CONFIG}_${BIDDER_VERSION}`,
-    auid: '540141567',
+    auid: bids[0].params.unit,
```
Comment (bretg) [inline modules/openxoutstreamBidAdapter.js]: Why are you only looking at bidRequest[0] in this code? Sounds like there's an underlying assumption that outstream adunits will be in their own separate auctions apart from other adunits?

Comment (HolzAndrew) [reply modules/openxoutstreamBidAdapter.js]: @bretg I updated the adapter to make multiple requests now

File: modules/openxoutstreamBidAdapter.js [hunk 2]
```diff
@@ -110,24 +103,28 @@ function buildCommonQueryParamsFromBids(bids, bidderRequest) {
   return defaultParams;
 }
 
-function buildOXBannerRequest(bids, bidderRequest) {
-  let queryParams = buildCommonQueryParamsFromBids(bids, bidderRequest);
-  queryParams.aus = utils._map(bids, bid => utils.parseSizesInput(bid.sizes).join(',')).join('|');
+function buildOXBannerRequest(bid, bidderRequest) {
+  let queryParams = buildCommonQueryParamsFromBids(bid, bidderRequest);
+  // let auids = utils._map(bids, bid => bid.params.unit)
```
Comment (jaiminpanchal27) [inline modules/openxoutstreamBidAdapter.js]: Can you remove this commented code

**Comment by jimee02** [comment]
LGTM

**Comment by HolzAndrew** [comment]
@jaiminpanchal27 it looks like a test unrelated to our adapter has failed. Let me know if there is anything you need from us to update.
Thanks!

screen shot 2018-10-29 at 3 38 26 pm

**Pull request merged**